lib/modules/datasource/maven/index.spec.ts

```diff
@@ -719,13 +713,22 @@ describe('modules/datasource/maven/index', () => {
         s3mock.reset();
       });
 
+      function body(input: string) {
```

Comment:
doesn't typescript or eslint complain bout missing return type?

Reply:
I think TS can imply return types now?

Reply:
In the tests I believe it doesn't